lib/ret/hub.ex

```diff
@@ -50,6 +50,13 @@ defmodule Ret.Hub do
     pin_objects: true
   }
 
+  @default_restrictive_member_permissions %{
```

Comment:
ah this is an interesting approach! i was thinking we would expose all the permissions individually as on/off switches, but i think the approach you've taken here may be smarter, because it means as we add more permissions we can 'do the right thing' based upon the users meta-preference of if rooms should be 'open' or 'closed'. i think given our current permissions, it seems unlikely that a mixed default (eg media but no pen) probably isn't worth supporting in lieu of this.

Reply:
Yeah I prefer this approach, especially since there is logic involved with the current permission settings -- namely that you cannot toggle `spawn_camera` and `pin_objects` independently of `spawn_and_move_media`.